pyproject.toml

```diff
@@ -12,7 +12,7 @@ boto3 = "^1.34.119"
 datahub = "^0.999.1"
 moto = "4"
 pandas = "^2.2.2"
-acryl-datahub = { extras = ["dbt"], version = "~=0.13.3.4" }
+acryl-datahub = { extras = ["dbt"], version = "==0.13.2.4" }
```

Comment:
is the downgrading here intentional?

Reply:
This should probably be reverted and applied via a different PR. The `0.13.3.X` ingestions are [failing with an ingestion error on `main`](https://mojdt.slack.com/archives/C03QZ776JVA/p1721747684834469?thread_ts=1721745324.616259&cid=C03QZ776JVA)

Reply:
Yeah I think a separate PR would be good, makes it easier to roll back either part if we need to.

Do we have a ticket to upgrade again and fixt how we create the Domain entities?